apps/studio/src/components/AvatarMenu.tsx

```diff
@@ -0,0 +1,35 @@
+import { Divider, Flex, Text } from "@chakra-ui/react"
+import {
+  AvatarMenu as AvatarMenuComponent,
```

Comment:
nit: Can name this as `OgpAvatarMenu` since technically this file is also a component.

Reply:
updated in https://github.com/opengovsg/isomer/pull/999/commits/9182d9528711b32625c17ba909bb934a621161be